### PR TITLE
9.0 add date range Sorrento

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by KevinBro** [comment]
There is a missing bracket at the bottom of the date_range.js file : })

File: date_range/models/date_range.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                SELECT
+                    id
+                FROM
+                    date_range dt
+                WHERE
+                    DATERANGE(dt.date_start, dt.date_end, '[]') &&
+                        DATERANGE(%s::date, %s::date, '[]')
+                    AND dt.id != %s
+                    AND dt.active
+                    AND dt.company_id = %s
```
Comment (sbidoul) [inline date_range/models/date_range.py:60]: this breaks if company_id is False

**Comment by lmignon** [comment]
@KevinBro Thank you for the report. It's fixed.

File: date_range/models/date_range.py (hunk 1 above)
Comment (lmignon) [inline date_range/models/date_range.py:60]: @sbidoul Thank you fixed in     baa85c0

**Comment by jgrandguillaume** [comment]
LGTM :+1:

**Comment by lmignon** [comment]
Review continue on OCA https://github.com/OCA/server-tools/pull/417

**Pull request closed without merging**